.pylintrc

```diff
@@ -424,6 +424,7 @@ confidence=HIGH,
 disable=too-many-lines,
         too-many-arguments,
         too-many-instance-attributes,
+        import-error,
```

Comment:
My thinking was that this only gets raised when a file tries to import something that isn't installed, and if this is because we're actually missing a requirement then the test will catch them and fail anyway.  I could imagine a scenario in which a fresh repo is cloned and not all of the dev requirements are installed, and pylint would then complain!